tests/Microsoft.Identity.Test.Android.UIAutomation/androidTests.cs

```diff
@@ -70,12 +70,10 @@ public void RunAllTests()
                 AcquireTokenADFSV2019InteractiveNonFederatedTest,
 
                 B2CLocalAccountAcquireTokenTest,
-                B2CROPCLocalAccountAcquireTokenTest
-                // Google Auth does not support embedded webview from b2clogin.com authority.
-                // App Center cannot run system browser tests yet, so this test can only be run in system browser locally.
-                //B2CGoogleB2CLoginAuthorityAcquireTokenTest,
-
+                B2CROPCLocalAccountAcquireTokenTest,
```

Comment:
Is there any code path covered by this test that is not covered by the similar test you wrote for desktop? AFAIK ROPC is identical between all platforms where it is enabled, the only risks here are that we disable the ROPC entry point on mobile.

Reply:
we have this test on desktop and in integration tests. I do see value in having it on mobile as we have run into the issue where we broke scenario on iOS or android and the UI tests picked it up. Since it's not complicated, i don't see the hurt in having it.

---
In reply to: [296148945](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1226#discussion_r296148945) [](ancestors = 296148945)

Reply:
The hurt in having it is having to maintain it and the extra time it adds to the test run. But since you added it and it's low risk of causing issues, let's keep it.